### PR TITLE
Daisy Chains Java Solution || Purging Bronze of an Irrelevant Problem

### PR DESCRIPTION
**Pull request by sathu-beep**
Created a Daisy Chains Java Solution since that is missing from guide and that is much cleaner, easier to understand, and has a much much better editorial than the Official Solution which is only 2 lines 😭 

Deleted SIlver Problem Rest Stops from bronze "intro to greedy" since A) Nathan told me to remove unhelpful problems B) this is too hard to be in "intro to greedy" and C) its not even sorting so it cant go in the greedy sorting section in silver so it should begone

😏 team members write suggestions/feedback and also DONT nitpick over the formatting of simple unimportant things we all have our own ideas in our heads and I would prefer to only implement formatting that Nathan tells me to 👿 

also its no secret that I'm trying to get back onto team so I will continue making sols with editorial as soon as this is merged

### CODE REVIEW

File: solutions/usaco-1060.mdx [hunk 1]
```diff
@@ -53,5 +53,83 @@ int main(){
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+// Created by Sathvik Chundru with help from Justin Chen
```
Comment (SansPapyrus683) [inline solutions/usaco-1060.mdx:60]: i mean there's the author page, seems kinda unnecessary

File: solutions/usaco-1060.mdx [hunk 2]
```diff
@@ -53,5 +53,83 @@ int main(){
...
+import java.util.*;
+import java.io.*;
```
Comment (SansPapyrus683) [inline solutions/usaco-1060.mdx:62]:
```suggestion
import java.io.*;

```

File: solutions/usaco-1060.mdx [hunk 3]
```diff
@@ -53,5 +53,83 @@ int main(){
...
+						      
+		System.out.println(counter);
+		sc.close();
+	}
+
+}
+
+```
+				 
+</JavaSection>
```
Comment (SansPapyrus683) [inline solutions/usaco-1060.mdx:133]: lines 68, 70, 75, 80, 83, 86, 95, 97, 101, 107, 109, 112, 117, 128, 130, and 132 are unnecessary whitespace

File: solutions/usaco-1060.mdx [hunk 4]
```diff
@@ -53,5 +53,83 @@ int main(){
...
+
+				// average = total sum of petals / total number 
+				// of flowers
+
+				int avg = sum / flowerCount;
+
+				for (int k = i; k <= j; k++) {
+					if (petals[k] == avg) {
+
+						// checking to see if any flowers
```
Comment (SansPapyrus683) [inline solutions/usaco-1060.mdx:113]:
```java
/*
 * multiline comments should be styled
 * like this not with just a bunch of single line comments
 */
```

File: solutions/usaco-1060.mdx [hunk 5]
```diff
@@ -53,5 +53,83 @@ int main(){
...
+
+		int counter = 0;
+
+		// creating 2 nested arrays to brute force every 
+		// possible combination of flowers
+
+		for (int i = 0; i < n; i++) {
+			for (int j = i; j < n; j++) {
+
+				int flowerCount=j-i+1;
```
Comment (SansPapyrus683) [inline solutions/usaco-1060.mdx:87]: whitespace between operators please

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
formatting stuff

File: solutions/usaco-1060.mdx (hunk 1 above)
Comment (sathu-beep) [reply solutions/usaco-1060.mdx:60]: k @24justinch

Comment (sathu-beep) [reply solutions/usaco-1060.mdx:60]: is the guy who I credited. If I have to remove his name then at least let his name be immortalized here

File: solutions/usaco-1060.mdx (hunk 4 above)
Comment (sathu-beep) [reply solutions/usaco-1060.mdx]: implemented 😒

File: solutions/usaco-1060.mdx (hunk 5 above)
Comment (sathu-beep) [reply solutions/usaco-1060.mdx]: fixed shit i forgot that one spot
coding guy pls don't hold against me in my team app am sorry I tried my best to get everything

File: solutions/usaco-1060.mdx (hunk 3 above)
Comment (sathu-beep) [reply solutions/usaco-1060.mdx:127]: fixed
thought it looked better but sans is an :orz:

File: solutions/usaco-1060.mdx (hunk 2 above)
Comment (sathu-beep) [reply solutions/usaco-1060.mdx:61]: done

**Review by SansPapyrus683 — APPROVED** [review]
epic

File: solutions/usaco-1060.mdx [hunk 6]
```diff
@@ -2,7 +2,7 @@
 id: usaco-1060
 source: USACO Bronze 2020 December
 title: Daisy Chains
-author: Danh Ta Chi Thanh
+author: Sathvik Chundru [Java], Danh Ta Chi Thanh [CPP]
```
Comment (HarryWangATX) [inline solutions/usaco-1060.mdx:5]: remove the languages

**Review by HarryWangATX — CHANGES_REQUESTED** [review]

File: solutions/usaco-1060.mdx (hunk 6 above)
Comment (sathu-beep) [reply solutions/usaco-1060.mdx:5]: k

Comment (sathu-beep) [reply solutions/usaco-1060.mdx]: done can u approve or make another suggestion

File: solutions/usaco-1060.mdx [hunk 7]
```diff
@@ -53,5 +53,77 @@ int main(){
...
+		int n = sc.nextInt();
+		int[] petals = new int[n];
+		for (int i = 0; i < n; i++) {
+			/* 
+			 * storing each petal count in an array
+			 * so we can keep track of which flower
+			 * has how many petals based on the 
+			 * index it is stored in
+			 */
+			petals[i]=sc.nextInt();
```
Comment (maggie-j-liu) [inline solutions/usaco-1060.mdx:75]:
```suggestion
			petals[i] = sc.nextInt();
```

File: solutions/usaco-1060.mdx [hunk 8]
```diff
@@ -53,5 +53,77 @@ int main(){
...
+		}
+		int counter = 0;
+		/* 
+		 * creating 2 nested arrays to brute force every
```
Comment (maggie-j-liu) [inline solutions/usaco-1060.mdx:79]: arrays??

File: solutions/usaco-1060.mdx [hunk 9]
```diff
@@ -53,5 +53,77 @@ int main(){
...
+	
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+	
+class Main {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
```
Comment (maggie-j-liu) [inline solutions/usaco-1060.mdx:65]: scanner should be avoided https://usaco.guide/general/adding-solution?lang=java#java can you change it to kattio?

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: solutions/usaco-1060.mdx (hunk 9 above)
Comment (SansPapyrus683) [reply solutions/usaco-1060.mdx:65]: i mean it still get's ac'd w/o

File: solutions/usaco-1060.mdx (hunk 8 above)
Comment (SansPapyrus683) [reply solutions/usaco-1060.mdx:79]: actually maybe just replace this entire comment w/ `// brute force every possible subsequence of flowers`

File: solutions/usaco-1060.mdx (hunk 9 above)
Comment (HarryWangATX) [reply solutions/usaco-1060.mdx:65]: no scanners 👀

Comment (HarryWangATX) [reply solutions/usaco-1060.mdx:65]: also rename to `io` after changing to kattio

Comment (sathu-beep) [reply solutions/usaco-1060.mdx:65]: ye this is bronze theres no way scanner effects run time in bronze lmfao 

but again maggie _is_ content author so I should listen to her. I will change that thing if another person tells me to 👍

File: solutions/usaco-1060.mdx (hunk 8 above)
Comment (sathu-beep) [reply solutions/usaco-1060.mdx:79]: bruh i was doing math hw while doing the editorial 
ill fix it m bad

Comment (dongliu0426) [reply solutions/usaco-1060.mdx:79]: > actually maybe just replace this entire comment w/ `// brute force every possible subsequence of flowers`

`// brute force every possible subarray of flowers`

Comment (sathu-beep) [reply solutions/usaco-1060.mdx:79]: done

File: solutions/usaco-1060.mdx (hunk 9 above)
Comment (sathu-beep) [reply solutions/usaco-1060.mdx:65]: so is kattio this whole thing
```java
class Kattio extends PrintWriter {
	private BufferedReader r;
	private StringTokenizer st;
	// standard input
	public Kattio() { this(System.in,System.out); }
	public Kattio(InputStream i, OutputStream o) {
		super(o);
		r = new BufferedReader(new InputStreamReader(i));
	}
	// USACO-style file input
	public Kattio(String problemName) throws IOException {
		super(new FileWriter(problemName+".out"));
		r = new BufferedReader(new FileReader(problemName+".in"));
	}
	// returns null if no more input
	public String next() {
		try {
			while (st == null || !st.hasMoreTokens())
				st = new StringTokenizer(r.readLine());
			return st.nextToken();
		} catch (Exception e) {}
		return null;
	}
	public int nextInt() { return Integer.parseInt(next()); }
	public double nextDouble() { return Double.parseDouble(next()); }
	public long nextLong() { return Long.parseLong(next()); }
}
```
so I keep this thing in my code?

Comment (maggie-j-liu) [reply solutions/usaco-1060.mdx:65]: Use the kattio [codesnip](https://usaco.guide/general/working-mdx#kattio)

Comment (sathu-beep) [reply solutions/usaco-1060.mdx:65]: "kattio" more like "catgirl owo" isnt that right @SansPapyrus683

Comment (sathu-beep) [reply solutions/usaco-1060.mdx:65]: @maggie-j-liu thanks for the snip ill look into it later tonight when I get the chance

Comment (SansPapyrus683) [reply solutions/usaco-1060.mdx:65]: > "kattio" more like "catgirl owo" isnt that right @SansPapyrus683

you better sleep with one eye open buddy

Comment (sathu-beep) [reply solutions/usaco-1060.mdx:65]: ez

Comment (HarryWangATX) [reply solutions/usaco-1060.mdx:65]: can we not lmao.

Comment (sathu-beep) [reply solutions/usaco-1060.mdx]: done kattio... it doesnt come in github so all it says is "//CodeSnip{Kattio}	" but when you paste my code into usaco guide editor it works 
😄

File: solutions/usaco-1060.mdx [hunk 10]
```diff
@@ -53,5 +53,75 @@ int main(){
...
+						 * petals as the average; if they do, 
+						 * then adding them to a counter
+						 */
+						counter++;
+						break;
+					}
+				}
+			}
+		}				      
+		System.out.println(counter);
```
Comment (dongliu0426) [inline solutions/usaco-1060.mdx:117]: use kattio :D

Comment (sathu-beep) [reply solutions/usaco-1060.mdx:117]: ok

Comment (sathu-beep) [reply solutions/usaco-1060.mdx:117]: it doesnt have a printing method tho

Comment (SansPapyrus683) [reply solutions/usaco-1060.mdx:117]: `io.println`?

Comment (sathu-beep) [reply solutions/usaco-1060.mdx]: done

**Review by maggie-j-liu — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Comment by sathu-beep** [comment]
@maggie-j-liu @HarryWangATX @dongliu0426 
nathan told me to tell one of the content managers to merge

**Comment by HarryWangATX** [comment]
okay let me look over it

**Review by HarryWangATX — APPROVED** [review]
